client/apps/edit/components/content/sections/text/index.coffee

```diff
@@ -48,6 +48,7 @@ module.exports = React.createClass
     @hasFeatures = @hasFollow
 
   componentDidMount: ->
+    @props.sections.on 'change:autolink', @editorStateFromProps
     if @props.section.get('body')?.length
```

Comment:
Sorry should have realized this earlier -- we are deprecating `client/apps/edit/components/content/` in favor of `/content2` very soon, is there a way to port this into the new text component? Maybe in a later PR?

Reply:
Yep I am aware and working on that part now :)

Reply:
awesome!